README.md

```diff
@@ -192,6 +194,17 @@ curl -X PUT http://127.0.0.1:4567/v1/auth/perms/sample-user -d '{
 }'
 ```
 
+Queue group permissions. Here `bar.>` is the subject and `fizzgroup` is the
```

Comment:
Just reading it, I'd say something like `Queue group permissions are supported as well.  Here...`